lib/vanagon/component.rb

```diff
@@ -265,7 +265,8 @@ def fetch_url(options)
     # a rewritten URL if there's any value for #url.
     def deprecated_rewrite_url
       return nil unless url
-      Vanagon::Component::Source::Rewrite.parse_and_rewrite(url)
+      rewritten_url = Vanagon::Component::Source::Rewrite.parse_and_rewrite(url)
```

Comment:
Maybe add a comment here explaining what's up?

Reply:
I updated both the comment for this method and the docstring for the `mirrors` accessor